app/views/courses/gcse_requirements/edit.html.erb

```diff
@@ -38,8 +38,9 @@
             <% if @course.is_primary? %>
               <%= f.govuk_check_box :accept_science_gcse_equivalency, "Science", label: { text: "Science" }, data: { qa: "gcse_requirements__science_equivalency" } %>
             <% end %>
-            <%= f.govuk_text_area :additional_gcse_equivalencies, label: { text: "Details about equivalency tests you offer or accept", size: "s" }, hint: { text: "For example, if you offer the tests or ask candidates to use a third party, and if there are any costs to pay." }, data: { qa: "gcse_requirements__additional_requirements" } %>
+            <br>
```

Comment:
We shouldn’t need this `<br>` – spacing can be provided by wrapping the fieldset in a `<div class="govuk-form-group"></div>` (see prototype).

Reply:
> spacing can be provided by wrapping the fieldset in a `<div class="govuk-form-group"></div>`

…which is what the formbuider providers. So just remove the `<br>` 😄

Reply:
@paulrobertlloyd & @frankieroberto I think this was in a slightly differnt place until rubocop shifted it! will remove!